content/src/licenses/homemaker-home-health-aide.md

```diff
@@ -3,8 +3,8 @@ notesMd: >-
   https://dev.azure.com/NJInnovation/Business%20First%20Stop/_workitems/edit/4227
 
 
-  Renew My Health Care Services Firm Registration 
-urlSlug: healthcare
+  Renew My Health Care Services Firm Registration
```

Comment:
[dust] Is this missing a `title:`?

Reply:
Good eye. I looked into this, it's not an issue. There is no title field, this is part of the notes.